tree/ntuple/v7/inc/ROOT/RFieldBase.hxx

```diff
@@ -740,7 +741,7 @@ public:
    /// Reads 'size' values from the associated field, starting from 'firstIndex'. Note that the index is given
    /// relative to a certain cluster. The return value points to the array of read objects.
    /// The 'maskReq' parameter is a bool array of at least 'size' elements. Only objects for which the mask is
-   /// true are guaranteed to be read in the returned value array.
+   /// true are guaranteed to be read in the returned value array. A 'nullptr' means to read all elements.
```

Comment:
This is different than what we originally discussed: I first implemented that `maskReq` is mandatory and then omit it as an optimization in `ReadBulk(RNTupleClusterRange)`. However, this felt weird - why would we allow something for ourselves, but not for our users... But if wanted, I can easily change it back